spec/controllers/projects_controller_spec.rb

```diff
@@ -107,6 +96,7 @@
         get :show, params: { id: project.to_param }
         expect(response).to be_successful
         expect(response.body).to include('Number of volunteers')
+        expect(response.body).to include('Sign up to volunteer')
```

Comment:
I've moved the tests for a sign-up button to the show page as the button is already in existence and I think this is where we would want users to click sign-up:
![image](https://user-images.githubusercontent.com/27900457/81061757-d10d3e00-8ecc-11ea-95a7-40d541f33e55.png)